src/errorhandling/ErrorHandler.ts

```diff
@@ -29,46 +29,29 @@ export class ErrorHandler {
    */
   public static extrinsicFailed(extrinsicResult: SubmittableResult): boolean {
     const events: EventRecord[] = extrinsicResult.events || []
-    return (
-      events.find((eventRecord: EventRecord) => {
-        return (
-          !eventRecord.phase.asApplyExtrinsic.isEmpty &&
-          eventRecord.event.index.toHex() === SystemEvent.ExtrinsicFailed
-        )
-      }) !== undefined
-    )
-  }
-
-  public constructor(apiPromise: ApiPromise) {
```

Comment:
If you removed the constructor. 

Why should this be a class now? 

Shouldn't we just have these as functions?

Reply:
you're right, will change